tests/Aspire.Hosting.Azure.Tests/AzureStorageExtensionsTests.cs

```diff
@@ -231,10 +231,11 @@ public async Task AddBlobContainer_ConnectionString_resolved_expected_RunAsEmula
         var blobs = storage.AddBlobs("blob");
```

Comment:
Can we also update this test to verify blob containers work end-to-end?

https://github.com/dotnet/aspire/blob/69143ae9e72a93da8bdd8627e9b516f8f633faf9/tests/Aspire.Hosting.Azure.Tests/AzureStorageEmulatorFunctionalTests.cs#L109-L134

Reply:
I don't understand, update `AddBlobContainer_ConnectionString_resolved_expected_RunAsEmulator` or `VerifyAzureStorageEmulatorResource`. The first one is a unit test, the second is already testing the container client with the emulator.

Reply:
Update the `VerifyAzureStorageEmulatorResource` functional e2e test to also ensure that Hosting `builder.AddBlobContainer("container");` and Client side `builder.AddAzureBlobContainerClient("container");` work together end-to-end.